java/src/client/game/GameManager.java

```diff
@@ -54,6 +54,7 @@
      * @post This provides valid operations on GameManager
      */
     GameManager() {
+        clientModel = new ClientModel();
```

Comment:
Wouldn't it make more sense for this to initially be null if the game hasn't started yet? If the client code can't handle it that's a fault of the client code. That way we catch the problem areas more quickly.